module/Finna/src/Finna/RecordDriver/SolrForward.php

```diff
@@ -276,6 +297,14 @@ public function __construct(
         $searchSettings = null
     ) {
         parent::__construct($mainConfig, $recordConfig, $searchSettings);
+        if (strpos($_SERVER['HTTP_HOST'], "finna-pre") !== false) {
```

Comment:
Ei voi testata näin. Joku asetus tms. ennemmin, mutta miksi edes tarvitaan kaksi URLia, kun testauksessa voi käyttää finna-pre-fe:llä voi määritellä toisen ja tuotannossa toisen, jos tarvitsee?